components/list/demo/simple.md

```diff
@@ -15,7 +15,11 @@ title:
 
 ## en-US
 
-Simple List.
+Ant Design supports a default list size as well as a large and small size.
+
+If a large or small list is desired, set the size property to either large or small respectively. Omit the size property for a list with the default size.
+
+Otherwise to setting `header` and `footer` property can custom header and footer of list.
```

Comment:
=> You can also customize the header and footer of list by setting `header` and `footer` property.